### PR TITLE
Indaba-922: weighted box

### PR DESCRIPTION
**Pull request by morleegan**
#### What does this PR do?
Fixes the CSS to properly keep box inside question block.

#### Related JIRA tickets:
https://jira.amida.com/projects/INBA/issues/INBA-922

#### How should this be manually tested?
check to see if the weighted boxes appear outside the question block in survey builder. 

#### Screenshots (if appropriate):
screen shot 2019-01-07 at 12 06 58 pm

### CODE REVIEW

File: src/styles/surveybuilder/_dynamic-question.scss [hunk 1]
```diff
@@ -2,7 +2,7 @@ $block-class: 'dynamic-question';
 
 @at-root {
     .#{$block-class} {
-        padding: 0 1em 0.5em;
+        padding: 0 2em 1em;
```
Comment (jmfadeley) [inline src/styles/surveybuilder/_dynamic-question.scss]: So I'm not sure if increasing this padding at root is a good answer. It creates something of an indent for the options. This indention appears in FIC (Firefox, IE, Chrome). 
image

Worse, at least for me it sometimes creates bounciness when you flip the "Weighted Value Question" on and off in Chrome (oddly, I don't see this in IE or Firefox). Try it and you may see the right side of the panel shift just a few pixels. If not, let me know.

A possible solution I'd recommend is to set the padding to what it was (or less, say `padding: 0 1em` and eliminate the `0.5em` padding for the bottom). Then, under `dynamic-question__choices__input` (line 24 of `_dynamic-question.scss`), I would adjust the `width` to be `47vw` or `48vw` (it currently stands at `50vw`). This eliminates the bounciness and lets the boxes appear cleanly. 

I would check with @laceyIrvin if this is ideal or if she wants to line up with the grey question box above it.

Comment (laceyIrvin) [reply src/styles/surveybuilder/_dynamic-question.scss]: `@@ -3,10 +3,9 @@ $block-class: 'question-panel';
 @at-root {
     .#{$block-class} {
         border: 2px solid $border-color;
-        padding: 8px;
-        margin-bottom: 24px;
         border-radius: 2px;
         background-color: #fff;
+        padding: 1em;
 
         &__header {
             font-size: 1.3em;
@@ -17,8 +16,7 @@ $block-class: 'question-panel';
         &__question {
             display: flex;
             flex-direction: column;
-            padding: 8px;
-            width: 100%;
+            width: 75%;
 
             &-number {
                 @extend .small-font;
@ $block-class: 'question-panel';
             &-text {
                 color: #000;
                 font-size: 16px;
-                overflow: hidden;
                 background-color: #fafafa;
-                resize: none;
                 outline: none;
                 border: 1px solid $border-color;
                 border-radius: 2px;
-                padding: 8px;
+                padding: .5em;
 
                 &:focus,
                 &:active {
$block-class: 'question-panel';
             display: flex;
             justify-content: space-between;
             border: none;
-            padding-top: 8px;
-            padding-left: 8px;`

Comment (laceyIrvin) [reply src/styles/surveybuilder/_dynamic-question.scss]: @morleegan Do you want me to push up my changes I made locally?
screenshot 2019-01-08 16 04 24

**Comment by morleegan** [comment]
@jmfadeley @eawest2 - Lacey and Riley worked on this ticket yesterday. Check out the edits and retest. I looked through it and it looks good but, I cannot approve it bc I created the pull

File: src/styles/surveybuilder/_dynamic-question.scss (hunk 1 above)
Comment (morleegan) [reply src/styles/surveybuilder/_dynamic-question.scss]: @laceyIrvin - come back in and approve your PR. I checked out the scroll and box enlargement feature. it looks good

**Comment by eawest2** [comment]
I found a break point where the weighted boxes still fall outside the question block in chrome. It seems to be some sort of edge case between two styling break points. 

It looks like the window aspect is about 580px wide when it does this. 

image

**Pull request merged**